backend/prompts/rest_prompt.py

```diff
@@ -33,7 +33,7 @@ def get_api_analysis_prompt() -> BasePrompt:
     @staticmethod
     def get_basic_test_prompt() -> BasePrompt:
         """Prompt for generating basic RestAssured tests"""
-        template = """En tant qu'ingénieur de test API expérimenté, génère un test RestAssured complet pour l'API Spring Boot suivante. Le test doit suivre les meilleures pratiques et inclure toutes les validations nécessaires.
+        template = """En tant qu'ingénieur de test API expérimenté, génère un test RestAssured complet pour l'API Spring Boot suivante, en inclure toutes les validations nécessaires.
```

Comment:
TBD if this really helps. But at least I'd make it gramatically correct
```suggestion
        template = """En tant qu'ingénieur de test API expérimenté, génère un test RestAssured complet pour l'API Spring Boot suivante. Inclus toutes les validations nécessaires.
```